Content.Shared/Standing/StandingStateSystem.cs

```diff
@@ -101,7 +101,9 @@ public bool IsDown(EntityUid uid, StandingStateComponent? standingState = null)
                 }
             }
 
-            if (!_gameTiming.IsFirstTimePredicted)
+            // check if component was just added or streamed to client
+            // if true, no need to play sound - mob was down before player could seen that
+            if (!_gameTiming.IsFirstTimePredicted || standingState.LifeStage <= ComponentLifeStage.Starting)
```

Comment:
Should be fine to remove IsFirstTimePredicted I think as PlayPredicted already checks it.